olm/prepare-community-operators-update.sh

```diff
@@ -121,9 +121,11 @@ do
   > "${folderToUpdate}/${lastPackagePreReleaseVersion}/eclipse-che.v${lastPackagePreReleaseVersion}.clusterserviceversion.yaml"
 
   echo
-  echo "   - Update the CRD file"
+  echo "   - Update the CRD files"
   cp "${packageBaseFolderPath}/${lastPackagePreReleaseVersion}/org_v1_che_crd.yaml" \
   "${folderToUpdate}/${lastPackagePreReleaseVersion}/checlusters.org.eclipse.che.crd.yaml"
+  cp "${packageBaseFolderPath}/${lastPackagePreReleaseVersion}/org.eclipse.che_checlusterbackups_crd.yaml" "${folderToUpdate}/${lastPackagePreReleaseVersion}/org.eclipse.che_checlusterbackups_crd.yaml"
+  cp "${packageBaseFolderPath}/${lastPackagePreReleaseVersion}/org.eclipse.che_checlusterrestores_crd.yaml" "${folderToUpdate}/${lastPackagePreReleaseVersion}/org.eclipse.che_checlusterrestores_crd.yaml"
```

Comment:
naming convention is different. Let's have something common.
`checlusters.org.eclipse.che.crd.yaml`
`org.eclipse.che_checlusterbackups_crd.yaml`
`org.eclipse.che_checlusterrestores_crd.yaml`

Reply:
I'd leave this improvement for a refactoring PR